<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>信息管理系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>411</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>301</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>学生详细信息</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>51</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>成绩：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>班级：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>出生省份：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>出生日期：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>性别：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>51</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>姓名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dbEditName">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>51</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>学号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dbComboSex">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="dbEditBirth">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="dbComboProvince">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>240</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="dbComboDep">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>290</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="dbSpinSalary">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>340</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="dbSpinEmpNo">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>181</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>251</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>学生头像</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="dbLabPhoto">
       <property name="text">
        <string>点击工具栏设置头像</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupboxs">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>310</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>学生介绍</string>
    </property>
    <widget class="QPlainTextEdit" name="dbEditMemo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>251</width>
       <height>171</height>
      </rect>
     </property>
     <property name="centerOnScroll">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>301</width>
      <height>28</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>查询姓名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="findName"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpenDB"/>
   <addaction name="actAdd"/>
   <addaction name="actInsert"/>
   <addaction name="actDelete"/>
   <addaction name="actPhoto"/>
   <addaction name="actSave"/>
  </widget>
  <action name="actOpenDB">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/new/prefix1/img/start.png</normaloff>:/new/prefix1/img/start.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="iconText">
    <string>打开</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actAdd">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/new/prefix1/img/add.jpg</normaloff>:/new/prefix1/img/add.jpg</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actInsert">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/new/prefix1/img/inserts.jpg</normaloff>:/new/prefix1/img/inserts.jpg</iconset>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actDelete">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/new/prefix1/img/delete.png</normaloff>:/new/prefix1/img/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actPhoto">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/new/prefix1/img/set.jpg</normaloff>:/new/prefix1/img/set.jpg</iconset>
   </property>
   <property name="text">
    <string>设置照片</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actSave">
   <property name="icon">
    <iconset resource="rec.qrc">
     <normaloff>:/new/prefix1/img/save.jpg</normaloff>:/new/prefix1/img/save.jpg</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rec.qrc"/>
 </resources>
 <connections/>
</ui>
